internal/controllers/machine/machine_controller_test.go

```diff
@@ -2559,10 +2603,11 @@ func TestIsDeleteNodeAllowed(t *testing.T) {
 				},
 				Status: clusterv1.MachineStatus{},
 			},
+			extraMachines: nil,
 			expectedError: errNilNodeRef,
 		},
 		{
-			name: "no control plane members",
+			name: "no control plane members, deleting a worker",
```

Comment:
Please add a comment reminding that this is a use case that might happen when CP is not implemented using Cluster API machines

Reply:
Sounds useful, added, thanks.